src/modules/tenant/service.js

```diff
@@ -67,19 +67,39 @@ export default class TenantService extends Service {
 
 	}
 
-	// get_current_tenant() method is used for obtaining current tenant
-	get_current_tenant() {
+	/*
+		getTenantData() method is used for returning the tenant data
+		It requires SeaCat Auth service
+	*/
+	async getTenantData() {
+		let tenantData = {};
+		let currentTenant = this.getCurrentTenant();
+		if (currentTenant) {
+			const SeaCatAuthAPI = this.App.axiosCreate('seacat_auth');
```

Comment:
This call can be cached, tenant data are quite static.
On the other hand, ETag-based caching can be quite OK here.

Reply:
Not sure if ETag would help here much, but I can take a look on that deeper if needed

Reply:
Please add "TODO" here: "Consider caching tenant data to reduce number of calls to the backend"

Reply:
@ateska tenant data are being cached as per refactorisation in this commit: https://github.com/TeskaLabs/asab-webui/commit/efb8e56d03cee6b7f5e5807390df60e97197ec30 , so I suppose there is no need to add that TODO, unless you wanted it for the `ETag`